examples/point-cloud-laz/app.js

```diff
@@ -124,8 +133,29 @@ class Example extends PureComponent {
   }
 
   _onUpdate() {
-    const {rotating, viewport} = this.state;
+    const {vrEnabled} = this.state;
+
+    if (vrEnabled) {
+      const {vrDisplay, emulatedPose} = this.state;
+      // animate camera in Z-axis
+      // this can be removed after we have a VRController for EmulatedVRDisplay
+      if (vrDisplay.isEmulated) {
+        const {position} = emulatedPose;
+        position[2] += position[2] < 0.5 ? 0.001 : -0.5;
+        this.setState({
+          emulatedPose: {
+            ...emulatedPose,
+            ...position
+          }
+        });
+      }
+
+      this.forceUpdate(); // explicitly refresh state; removing this breaks frame update in VR
+      window.requestAnimationFrame(this._onUpdate);
```

Comment:
You need to call `requestAnimationFrame` on the display, otherwise it won't work.

Have you tested this with any actual headset?